### PR TITLE
Mention that Doctrine does not use Entities public API

### PR DESCRIPTION
**Pull request by JarJak**
As discussed in https://github.com/doctrine/DoctrineBundle/issues/778

### CODE REVIEW

**Review by localheinz — COMMENTED** [review]
Maybe worth mentioning that entity constructors are also never invoked?

**Comment by JarJak** [comment]
@localheinz updated

File: docs/en/reference/working-with-objects.rst [hunk 1]
```diff
@@ -25,6 +25,14 @@ Work that have not yet been persisted are lost.
...
     during that request being lost.
 
+.. note::
+
+    Doctrine does NEVER touch the public API of methods in your entity 
+    classes (like getters and setters) nor the constructor method.
+    Instead, it uses reflection to set/get data from your objects.
+    When Doctrine fetches data from DB and saves it back,
+    any code put in your get/set methods won't be implicitly taken into account.
+
```
Comment (jwage) [inline docs/en/reference/working-with-objects.rst]: Remove the extra new line here. We only need 1 new line in-between sections.

**Comment by JarJak** [comment]
@jwage updated

**Review by jwage — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]
:ship: :+1: :100:

**Pull request merged**